### ISSUE TITLE
[BUG] NHQM_v0.5.4.0_RC closes HWiNFO and FanControl

### ISSUE THREAD

**Issue by InterClaw** [nicehash/NiceHashQuickMiner#813]
**Describe the bug**
When starting NHQM_v0.5.4.0_RC, if HWiNFO is running or FanControl (https://github.com/Rem0o/FanControl.Releases). or both for that matter, these processes shut down. I'm not sure if this is intended behavior or not, i.e. that it's a feature of NHQM to shut down processes like this that could be interfering with OCTune. Or if this is a result of these processes fighting over the same resources. I added "-g" to the command line of NHQM, but it seemingly had no effect on this behavior.

**To Reproduce**
1. Launch HWiNFO and/or FanControl
2. Launch NHQM_v0.5.4.0_RC
3. Wait a second of two
4. Watch as HWiNFO and FanControl are shut down

**Expected behavior**
HWiNFO and FanControl are not shut down and can operate in parallel with NHQM_v0.5.4.0_RC.

**Screenshots**

**Version affected (please complete the following information):**
 - 0.5.4.0
 - RC

**NVIDIA driver version**
512.59 (but it was the same on the previous one)

**Hardware**
MSI GeForce RTX 3080 GAMING Z TRIO 10G LHR
AMD Ryzen 5 5600X

**Logs**
[log.txt](https://github.com/nicehash/NiceHashQuickMiner/files/8645420/log.txt)

**Additional context**

**Comment by InterClaw** [comment]
OpenHardwareMonitor does the same thing.

**Comment by kumoilain** [comment]
You may try HwInfo32, v7.06-4500 won't be closed on my PC when NHQM_RC is running.
My OS is win10 21h2.
FanControl will crash, and can't be restarted.

**Comment by mikeroygray** [comment]
Aquasuite's background service also gets closed. (It's a monitoring service aquasuite uses to control fans in water cooled systems.) This is particularly aggravating since the only values I need are air temps and water temps - none of which even involve the mb, let alone the gpu. But the service DOES look at that stuff, and apparently, just attempting to look at gpu values is causing the service to shut down.

**Comment by InterClaw** [comment]
ASRock's "Phantom Gaming Tuning" software still works to control the motherboard fans, but it's pretty clunky.

**Comment by InterClaw** [comment]
> You may try HwInfo32, v7.06-4500 won't be closed on my PC when NHQM_RC is running. My OS is win10 21h2. FanControl will crash, and can't be restarted.

Yep, can confirm that 32-bit 7.22 also works. Strange.

**Comment by moops44** [comment]
WinRing0x64.sys is the culprit in that case , 
try Shared Memory Support in the HWInfo Main Setting

**Comment by InterClaw** [comment]
> WinRing0x64.sys / IOMap64.sys is the culprit in that case , try Shared Memory Support in the HWInfo Main Setting

I don't think so. I'm already running with that on. I use it together with MSI Afterburner, FanControl, and Rainmeter.

**Comment by einkoro** [comment]
This also occurs with nvcp as well.

**Comment by kanadavos** [comment]
Also seeing the same behavior with EVGA Precision X1 and Open Hardware Monitor.

**Comment by SirDalavar** [comment]
THis guys fix solved my similar problems, i suspect it might work for you guys too.
https://www.reddit.com/r/NiceHash/comments/ukxoyh/fix_for_qmexcavator_054_rc_closing_hwmonitor_nvcp/

**Comment by mikeroygray** [comment]
> THis guys fix solved my similar problems, i suspect it might work for you guys too. https://www.reddit.com/r/NiceHash/comments/ukxoyh/fix_for_qmexcavator_054_rc_closing_hwmonitor_nvcp/

Nice - you beat me to it! Before I try the empty taskbar hack - do you think there's a REASON for this behavior?

**Comment by DevDocX** [comment]
> > You may try HwInfo32, v7.06-4500 won't be closed on my PC when NHQM_RC is running. My OS is win10 21h2. FanControl will crash, and can't be restarted.
> 
> Yep, can confirm that 32-bit 7.22 also works. Strange.

Another confirmation for 32-bit HWinfo.  Works and thanks.  Will do for now.

**Comment by InterClaw** [comment]
> THis guys fix solved my similar problems, i suspect it might work for you guys too. https://www.reddit.com/r/NiceHash/comments/ukxoyh/fix_for_qmexcavator_054_rc_closing_hwmonitor_nvcp/

Yeah, that worked. :) Okay, so this is not a bug it's a "feature" then.

Thanks but no thanks. NBMiner v41.0 does the same thing now, so it was fun to try out NHQM again, but not sticking around.

If the devs want, they can change the label of this ticket to something else then, like ranting about why this is a bad thing.

**Comment by mikeroygray** [comment]
They're may be more to this than meets the eye. The miner DOES run just
fine on my machine with the null .exe, but I also just got my first BSOD
when truning it off. Might have been a coincidence, but NH may have reasons
to lock out all other processes.

(Just guessing, though! I'm no programmer.)

Mike Gray, sent from my Huawei p30

**Comment by Dohtar1337** [comment]
A new release is coming with fixes. This bug report is obsolete.

If the issue repeats, please reopen a new issue.

Thank you for reaching out anyway!
